src/ml/neural_net/mps_layer_instance_norm_data_loader.mm

```diff
@@ -2,31 +2,31 @@
 #include <ml/neural_net/mps_weight.h>
 
 @interface TCMPSInstanceNormDataLoader () {
-  float *_gamma_weights;
-  float *_beta_weights;
+  NSMutableData *_gamma_weights;
+  NSMutableData *_beta_weights;
   
   NSString *_name;
   NSUInteger _styles;
 
-  id<MTLBuffer> _gammaBuffer;
-  id<MTLBuffer> _betaBuffer;
+  NSMutableArray<MPSVector *> *_gammaVectorArray;
+  NSMutableArray<MPSVector *> *_betaVectorArray;
+  NSMutableArray<MPSVector *> *_gammaMomentumVectorArray;
+  NSMutableArray<MPSVector *> *_gammaVelocityVectorArray;
+  NSMutableArray<MPSVector *> *_betaMomentumVectorArray;
+  NSMutableArray<MPSVector *> *_betaVelocityVectorArray;
```

Comment:
Instead of making all of these instance variables arrays, could we instead define one Objective C class to hold all of these things, and then create a single array of that?

Reply:
Note that it should suffice to just to do something like, in this .mm file:
```
@interface TCMPSMyClassNameGoesHere

@property (nonatomic) MPSVector *gammaMomentumVector;
@property (nonatomic) MPSVector *gammaVelocityVector;
...

@end

@implementation TCMPSMyClassNameGoesHere
@end  // All properties will be auto-synthesized
```

Then you can just use this like:
```
NSMutableArray<TCMPSMyClassNameGoesHere *> *styles;
for (NSUInteger index = 0; index < self.stylesCount; ++index) {
  TCMPSMyClassNameGoesHere *style = [[TCMPSMyClassNameGoesHere alloc] init];
  style.gammaMomentumVector = ...;
  ...;
}
```

Reply:
Ahh, that's a good idea!